readme.md

```diff
@@ -26,7 +26,7 @@ Tusk is [indicated by Evernote](https://help.evernote.com/hc/en-us/articles/2083
 
 You can now support the development process through [GitHub Sponsors](https://github.com/sponsors/klaussinani).
 
-Come over to [Gitter](https://gitter.im/klaussinani/tusk) or [Twitter](https://twitter.com/klaussinani) to share your thoughts on the project.
+Come over to [Gitter](https://gitter.im/klaussinani/tusk) or [𝕏 (Twitter)](https://twitter.com/klaussinani) to share your thoughts on the project.
```

Comment:
Now, twitter.com redirects to x.com, so the URL could be also get updated?

Reply:
Yes,

I just updated the URL